modules/ROOT/pages/access-developer-portal-task.adoc

```diff
@@ -1,25 +1,24 @@
-= Accessing the Developer Portal
+= Access the Developer Portal
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-API portals listed on an organization's developer portal that are public, or to which you have been granted Portals Viewer access, are visible to you. From the public Anypoint Platform Developer portal, you can access Anypoint Platform APIs to automate Anypoint Platform activities.
+As an administrator, you can view API portals if you have the appropriate access. The public https://anypoint.mulesoft.com/apiplatform/anypoint-platform/#/portals[Anypoint Platform developer portal], enables you to automate Anypoint Platform activities.
+ 
+You can view an API portal in Anypoint Platform if:
```

Comment:
One of the below permissions needs to be true.
Maybe 

> You can view an API portal in Anypoint Platform if either

Does it make sense?

Reply:
Done! Thanks!